### PR TITLE
[DO NOT MERGE] Editing: Remove snackbar when switching to html or visual mode

### LINKED ISSUE #11819: Snackbar notice when switching to HTML editor prevents editing last line of text
### Expected behavior
When I switch to the HTML editor it is easy for me to edit all areas of my post as I need to

### Actual behavior
When I switch to the HTML editor there is a pop up at the bottom of the screen preventing me from editing the last line of text in my post. This pop up lingers for a long time, preventing me from editing as I need to.

### Steps to reproduce the behavior
1. Create a new post with content in the block editor
2. Tap the 3 dots in the upper right corner to then tap switch to HTML
3. Try to edit the last line. You wont be able to until the pop up disappears, which can take awhile

##### Reported on Galaxy Note 8 WPAndroid 14.6.1
Ticket #2929016-zen

Example of hidden text:

(Image link: https://d.pr/i/tFtiRB )

(Image link: https://d.pr/i/nPWWFP )

### PR DESCRIPTION
**Pull request by ovitrif**
Fixes #11819

## Description
We received remarks that the snackbar notice shown when switching to the HTML editor prevents the user from being able to edit the last line(s) of text until it disappears.
This appears to be annoying for some users and doesn't seem to add any value.

On iOS or in the web version there is no such notification and the users can switch seamlessly between the Visual or the HTML editor mode.

It was suggested by @yaelirub to remove it and in a recent conversation suggested also by @antonis.

Given the above context I propose this solution, which:
- Removes the snackbar notice shown when switching to the HTML Mode in the editor, and
- Removes the snackbar notice shown when switching to the Visual Mode

## To test
1. From My Site press the Posts button or scroll down and select Blog Posts from the list
2. In post cards mode press the More button / In post list mode press the dots at the right of the post
3. Press Edit (first button)
4. In the editor, Tap the 3 dots in the upper right corner
5. Tap HTML Mode (3rd button)
6. **Verify** that no snackbar notice appears at the bottom of the screen
7. Tap again the 3 dots in the upper right corner
8. Tap Visual Mode  (3rd button)
9. **Verify** that no snackbar notice appears at the bottom of the screen

## Screenshots
<details>

 <summary>Switching to HTML mode</summary>

| Before | After |
| ----- | ----- |
| android_switch_to_html_before |  android_switch_to_html_after |
</details>

<details>

 <summary>Switching to Visual mode</summary>

| Before | After |
| ----- | ----- |
| android_switch_to_visual_before |  android_switch_to_visual_after |
</details>

## Regression Notes
1. Potential unintended areas of impact
  none

2. What I did to test those areas of impact (or what existing automated tests I relied on)
  manual testing

3. What automated tests I added (or what prevented me from doing so)
  n/a

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 18.7
 -----
+* [*] Editor: Removes snackbar notice when switching between HTML or Visual mode [https://github.com/wordpress-mobile/WordPress-Android/pull/15571]
```
Comment (antonis) [inline RELEASE-NOTES.txt:5]: Thank you for adding a release note on this 🙇 
Adding this comment to push it on the next release!

**Review by antonis — COMMENTED** [review]
Great work @ovitrif 👍 
I tested the app and it works as expected. The code changes also LGTM 🎉 
I'll wait for some [design feedback](https://github.com/wordpress-mobile/WordPress-Android/issues/11819#issuecomment-968296850) before approving the PR for merge.

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (ovitrif) [reply RELEASE-NOTES.txt:5]: Moved this to 18.8 👍🏼

**Comment by ovitrif** [comment]
Closing this pull-request since we're exploring a [different solution](https://github.com/wordpress-mobile/WordPress-Android/issues/11819#issuecomment-969670515) to use a non-obtrusive notice.

**Pull request closed without merging**